test/functional/fixtures/regression/gh-850/test.js

```diff
@@ -0,0 +1,9 @@
+describe('[Regression](GH-850)', function () {
+    it('Should resume the test if the cursor is moving from reloaded iframe to top window', function () {
+        this.timeout(40000);
```

Comment:
Let's rerun tests on this PR several times (3 is enought). If this test is ok, we will merge this. If it's unstable we should modify it or find the cause of this before merge